app/commands/slash_commands.rb

```diff
@@ -10,6 +10,7 @@ class SlashCommands
 
   def self.route
     router = Slash::Router.new
+    router.match match_regexp(/^$/), ActionCommand # lol hack this must be first to match first
```

Comment:
We should probably build a separate rack stack for handling the action requests. We can still use `lib/slash` as a framework since there's a lot of similarity between slash commands and actions.

I think, [here](https://github.com/Nosajool/slashdeploy/blob/2b82256421b45c69ba6dca8dc933ec906369b445/config/routes.rb#L12), we'd want to change to:

```diff
- mount SlashDeploy::Commands.slack, at: '/commands'
+ mount SlashDeploy.slack_commands, at: '/slack/commands'
+ mount SlashDeploy.slack_actions, at: '/slack/actions'
```

Then, we'd build a new rack stack for handling the actions:

```ruby
def slack_actions
  Slash::Rack.new(actions_handler)
end
```